src/prefect/logging/handlers.py

```diff
@@ -236,6 +237,44 @@ def _get_payload_size(self, log: Dict[str, Any]) -> int:
         return len(json.dumps(log).encode())
 
 
+class WorkerAPILogHandler(APILogHandler):
```

Comment:
It looks like we could have a `BaseAPILogHandler` class that this log handler and a `RunAPILogHandler` could subclass. I think that'd be cleaner and would enable a better structure for future log handlers.

Reply:
Can we kick this can down the road and it can be tackled later on?